apis/v1alpha1/iam_request.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+// IamRequest represents a request to update IAM policies.
+type IamRequest struct {
```

Comment:
For human-written code, the preferred name would be `IAMRequest`. Go is more tolerant with the code is auto-generated (which is why you'll see `Iam` in some client libraries).

Reply:
Good to know, updated.